### PR TITLE
Update minikf-gcp.md

### PR DESCRIPTION
**Pull request by jbottum**

### CODE REVIEW

File: content/en/docs/started/workstation/minikf-gcp.md [hunk 1]
```diff
@@ -2,16 +2,6 @@
 title = "Deploy using MiniKF on GCP"
 description = "Using Google Cloud Marketplace to deploy MiniKF (mini Kubeflow) on Google Cloud Platform (GCP)"
 weight = 40
-                    
-+++
```
Comment (alfsuse) [inline content/en/docs/started/workstation/minikf-gcp.md:6]: As per https://github.com/kubeflow/website/pull/2301 please add back this line to fix the deployment issue

File: content/en/docs/started/workstation/minikf-gcp.md [hunk 2]
```diff
@@ -2,16 +2,6 @@
 title = "Deploy using MiniKF on GCP"
 description = "Using Google Cloud Marketplace to deploy MiniKF (mini Kubeflow) on Google Cloud Platform (GCP)"
 weight = 40
```
Comment (8bitmp3) [inline content/en/docs/started/workstation/minikf-gcp.md:4]:
```suggestion
title = "Deploy using MiniKF on GCP"
description = "Using Google Cloud Marketplace to deploy MiniKF (mini Kubeflow) on Google Cloud Platform (GCP)"
weight = 40

+++
```

Comment (8bitmp3) [reply content/en/docs/started/workstation/minikf-gcp.md:4]: Hello @jbottum! As @alfsuse said, the deployment is failing - take a look at the suggested change that should fix it

File: content/en/docs/started/workstation/minikf-gcp.md [hunk 3]
```diff
@@ -2,16 +2,6 @@
 title = "Deploy using MiniKF on GCP"
 description = "Using Google Cloud Marketplace to deploy MiniKF (mini Kubeflow) on Google Cloud Platform (GCP)"
```
Comment (8bitmp3) [inline content/en/docs/started/workstation/minikf-gcp.md:3]:
```suggestion
title = "Deploy with MiniKF on Google Cloud"
description = "Using Google Cloud Marketplace to deploy MiniKF (mini Kubeflow) on Google Cloud"
```

- It was "deploy using MiniKF..." (title) vs "using ... Marketplace to deploy... " (description), so I'm suggesting a tweak here. LMKWYT 🙂
- Also, GCP has rebranded, I think

**Review by 8bitmp3 — CHANGES_REQUESTED** [review]
```
title = "Deploy with MiniKF on Google Cloud"
description = "Using Google Cloud Marketplace to deploy MiniKF (mini Kubeflow) on Google Cloud"
```

- It was "deploy using MiniKF..." (title) vs "using ... Marketplace to deploy... " (description), so I'm suggesting a tweak here. LMKWYT 🙂
- Also, GCP has rebranded, I think

**Comment by jbottum** [comment]
we will resubmit

**Pull request closed without merging**